firestore-bundle-server/POSTINSTALL.md

```diff
@@ -0,0 +1,76 @@
+# Using the extension
+When triggered by an HTTP request, this extension responds with a built data bundle stream.
+You can parameterize the request to stream data bundles from local cache; if bundle data is
+not available in local cache, queries will run against the Firestore back-end to get the
+data and build the bundle.
+
+If you use Firebase Hosting or Cloud Storage for Firebase in your projects, you can stream
+bundles from Hosting cache or a Cloud Storage bucket to your client-side apps. Firebase client
+SDKs provide a [data bundles API](https://firebase.devsite.corp.google.com/docs/firestore/bundles)
```

Comment:
Oops, domain wrong, my bad, this should be "firebase.google.com/docs/...."

Reply:
Fixed.